data-pipeline-operations/convert-xml-files-to-csv/convert-xml-files-to-csv-configuration.md

```diff
@@ -0,0 +1,259 @@
+---
+description: >-
+  This is the description of the JSON configuration file of a XML Conversion
+  data operation.
+---
+
+# XML Conversion configuration file
+
+The configuration file is in JSON format. It contains the following sections:
+
+* Global parameters: General information about the data operation.
+* Filenames templating: Optionally, you can add a creation step for a table that will contain the result of the extraction.
+* Credentials parameters : Mandatory credentials parameters of a service account that can access the input/output buckets.
```

Comment:
typo: no space needed between parameters and the : => "parameters: Mandatory"

Reply:
I see I couldn't modify the typo myself... I'm still not used to this review interface!